src/renderer/components/dock/create-resource.tsx

```diff
@@ -103,28 +108,40 @@ export class CreateResource extends React.Component<Props> {
 
     // skip empty documents
     const resources = yaml.loadAll(this.data).filter(Boolean);
-    const createdResources: string[] = [];
 
     if (resources.length === 0) {
       return void logger.info("Nothing to create");
     }
 
-    for (const result of await Promise.allSettled(resources.map(resourceApplierApi.update))) {
-      if (result.status === "fulfilled") {
-        createdResources.push(result.value.metadata.name);
-      } else {
-        Notifications.error(result.reason.toString());
+    const creatingResources = resources.map(async (resource: string) => {
+      try {
+        const data: KubeJsonApiData = await resourceApplierApi.update(resource);
+        const resourceName = data.metadata.name;
+        const resourceLink = apiManager.lookupApiLink({
+          kind: data.kind,
+          apiVersion: data.apiVersion,
+          name: resourceName,
+          namespace: data.metadata.namespace,
+        });
+
+        const showDetails = () => {
+          navigate(getDetailsUrl(resourceLink));
+          close();
+        };
+
+        const close = Notifications.ok(
+          <p>
+            Resource <a onClick={prevDefault(showDetails)}>{resourceName}</a> successfully created.
```

Comment:
maybe include the kind as well?

Reply:
Decided to put `kind` before `name` (looks better).
<img width="368" alt="Screenshot 2021-11-25 at 17 17 28" src="https://user-images.githubusercontent.com/6377066/143467104-2f9b2780-dda3-4b5d-9cf5-fad98b744a96.png">

Instead of:
<img width="370" alt="Screenshot 2021-11-25 at 17 16 42" src="https://user-images.githubusercontent.com/6377066/143467073-42b8ba97-9564-4b70-94f3-1518612a708e.png">